x-pack/plugins/cases/server/client/cases/find.ts

```diff
@@ -53,6 +53,20 @@ export const find = async (
     const { filter: authorizationFilter, ensureSavedObjectsAreAuthorized } =
       await authorization.getAuthorizationFilter(Operations.findCases);
 
+    /**
+     * Assign users to a case is only available to Platinum+
+     */
+
+    if (queryParams.assignees) {
```

Comment:
nit: `assignees` can be a single string I believe. I think that would result in this being false, should we check for an empty string to enter the if block too? I doubt that would happen often though.